worlds/banjo_tooie/Hints.py

```diff
@@ -0,0 +1,546 @@
+import enum
+import re
+from typing import Iterable, List, Set, Union, TYPE_CHECKING
+from dataclasses import dataclass
+from BaseClasses import ItemClassification, Location, LocationProgressType, CollectionState, MultiWorld
```

Comment:
```suggestion
from typing import Iterable, List, Union, TYPE_CHECKING
from dataclasses import dataclass
from BaseClasses import ItemClassification, Location, LocationProgressType, CollectionState
```
The unused imports refuse to die.